pkg/database/features.go

```diff
@@ -33,8 +33,24 @@ func WithCascading() Feature {
 	}
 }
 
-func WithOnSuccess(fn com.ProcessBulk[any]) Feature {
+func WithOnSuccess(fn ProcessBulk[any]) Feature {
 	return func(f *Features) {
 		f.onSuccess = fn
 	}
 }
+
+type ProcessBulk[T any] func(ctx context.Context, bulk []T) (err error)
```

Comment:
Can't all of this replaced with [OnSuccessSendTo](https://github.com/Icinga/icinga-go-library/blob/main/database/db.go#L417)?